libraries/ui-library/src/components/six-item-picker/six-item-picker.tsx

```diff
@@ -347,7 +320,7 @@ export class SixItemPicker {
       isNextItemAllowed: () =>
         maxLetter === undefined || this.getStringItemIndex(value) < this.getStringItemIndex(maxLetter),
       getNextItem: () => this.getNextItemByOperation(add),
-      getRoundtripItem: () => this.min,
+      getRoundtripItem: () => this.min as string,
```

Comment:
Same question as for max - `min` can also be a number, no? So wouldn't it be better to write () => String(this.min) if you always want to return a string?

Reply:
It should not be a number in this code path, but I think it is a good idea to use String(..) anyway -> Changed all ... as string to String(...)